tools/travis-test.sh

```diff
@@ -102,7 +105,7 @@ setup_chroot()
     python-dev python-nose python-pip cython"
 
   # faster operation with preloaded eatmydata
-  echo /usr/lib/libeatmydata/libeatmydata.so | \
+  echo '/usr/$LIB/libeatmydata.so' | \
```

Comment:
this is for inside the chroot, which is newer than trusty so always will use the multiarch path, when travis updates to xenial boxes the upper setting can be simplified too.
this setting should actually be done a bit earlier so we guarantee it is used for the dependency setup, done so now